source/extensions/filters/network/mongo_proxy/proxy.h

```diff
@@ -127,6 +127,7 @@ class ProxyFilter : public Network::Filter,
 
   // Network::WriteFilter
   Network::FilterStatus onWrite(Buffer::Instance& data, bool end_stream) override;
+  void initializeWriteFilterCallbacks(Network::WriteFilterCallbacks&) override{};
```

Comment:
I know this is not done for HTTP, but IMO it's fine to just have the interface have an empty definition for this function. Filters can override if needed. This will avoid needless implementation functions. Same comment applies elsewhere.

Reply:
Fixed